examples/calendar/backend/services/appointments/controller.ts

```diff
@@ -0,0 +1,6 @@
+import { NextFunction, Request, Response } from "express";
+import { User } from "../../models/user.model";
+
+export const create = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
this function is not doing anything. Do we need this?

Reply:
Left a comment on main thread, this is still work in progress, please review the completed tasks / existing functionality.

Reply:
please add a todo/WIP comment on the portions pending. Ideally WIP changes shall not be part of this PR. It creates confusion when reviewing changes